sam-styles/packages/components/button/styles/button.scss

```diff
@@ -4,6 +4,7 @@ $button-shadow: 0 0.25rem 0.5rem 0 rgba(0, 0, 0, 0.2);
   @include u-text("secondary-darker");
   @include u-border("primary");
   @include u-border-width("2px");
+  @include u-minw(15);
```

Comment:
It looks like this isn't resetting a style that was removed at some point, but adding a new style. Are you sure that a min-width should be added to **EVERY** button in SDS? Also what width does this convert to? Looks like around 100px minimum?

Reply:
Previously the min width was there but we removed it (in a different PR) because it was breaking the circle buttons and only added it to the pill buttons but in a parking lot meeting last week Christy also wanted the min-width of 120px on the regular buttons so I added it back and used min width 0 for the circle buttons.

Reply:
New requirement from Christy, got it. I wanted to make sure because it was not there prior to pill-button work, so wanted to make sure that a global change like that wasn't just making it's way in on accident.